src/js/angular/security/controllers.js

```diff
@@ -455,6 +455,16 @@ securityCtrl.controller('CommonUserCtrl', ['$rootScope', '$scope', '$http', 'toa
                 $scope.confirmPasswordError = '';
             }
         };
+
+        $scope.shouldDisableSameAs = function () {
+            const sameAsCheckbox = $('#sameAsCheck');
+            if ($scope.user && !$scope.user.appSettings.DEFAULT_INFERENCE && sameAsCheckbox.prop('checked')) {
+                sameAsCheckbox.prop("checked", false);
+                $scope.user.appSettings.DEFAULT_SAMEAS = false;
```

Comment:
Why are you modifying the default user settings, this does not look okay to me?

Reply:
In this way we force user to properly configure sameAs and include inferred. 
The same behaviour could be found on Visual and SPARQL views and was discussed with Pavel that we want the WB to behave consistent among different views.

Reply:
Ah, okay, that is exactly what my thoughts were. I didn't see that you make the change in the security page controller, I thought that you are adding such logic to the sparql view. My mistake. This is exactly how I thought it should work.